tests/python/unittest/test_tir_transform_lower_warp_memory.py

```diff
@@ -249,9 +245,13 @@ def check(m):
             B_np = A_np + 1
             tvm.testing.assert_allclose(B_nd.asnumpy(), B_np)
 
-    check(m=31)
-    check(m=32)
-    check(m=33)
+    for device in ['cuda', 'rocm']:
+        if not tvm.context(device, 0).exist or not tvm.runtime.enabled(device):
+            print("skip because", device,"is not enabled..")
+            continue
+        check(device, m=31)
```

Comment:
For rocm you may want to check 63,64,65 as it covers three cases comparing to the warp size.

Reply:
Tank you for your comments! I added both and rebased.